contracts/amm/MiltonInternal.sol

```diff
@@ -75,6 +73,8 @@ abstract contract MiltonInternal is
 
     uint32 internal _autoUpdateIporIndexThreshold;
 
+    mapping(address => bool) internal _swapLiquidators;
```

Comment:
how many liquidators we will have?
https://www.freecodecamp.org/news/how-to-implement-whitelist-in-smartcontracts-erc-721-nft-erc-1155-and-others/